sphinx_needs/layout.py

```diff
@@ -407,14 +429,21 @@ def _func_replace(self, section_nodes):
                         result = func(*func_args, **func_kargs)
 
                         if result:
-                            node_line += result
+                            if len(section_nodes) == 1:
+                                node_line = result
+                            else:
+                                node_line += result
                     else:
                         raise SphinxNeedLayoutException(
                             f"Error during layout line parsing. This looks strange: {line_element}"
                         )
 
                 return_nodes.append(node_line)
-        return return_nodes
+
+        if len(section_nodes) == 1:
```

Comment:
Ugly, but currently functional. I can refactor it, if we have an working solution.